Makefile.include

```diff
@@ -521,3 +521,19 @@ endif
 CFLAGS := $(patsubst -D%,,$(CFLAGS))
 CFLAGS := $(patsubst -U%,,$(CFLAGS))
 CFLAGS += -include '$(RIOTBUILD_CONFIG_HEADER_C)'
+
+# Set a custom channel if needed
+ifneq (,$(filter netdev2_ieee802154,$(USEMODULE)))
```

Comment:
Ok, for a common file, but I don't think it makes sense to put it in the examples folder, because it forces all application Makefiles to include it. It has an interest for example applications but not for people writing their own application outside of RIOT.
I would put this file at the root of RIOT code and include from the top Makefile.include if the build as a dependency on a network module (related to 802.15.4 or whatever is related to a DEFAULT_CHANNEL).